main.py

```diff
@@ -1,9 +1,23 @@
+import platform
 import pandas as pd
 import numpy as np
 import camelot
+import time
+from enum import Enum
+
+class SystemPlatform(Enum):
+    Windows = 1
+    Darwin = 2
+    Linux = 3
 
 # Read PDF Statement into a table collection, the areas/regions and columns separators is self-defined
-table = camelot.read_pdf(r'.\data\tng_ewallet_transactions.pdf', pages='all', flavor='stream',
+if platform.system() == SystemPlatform.Darwin.name or SystemPlatform.Linux.name:
```

Comment:
```suggestion
if platform.system() == SystemPlatform.Darwin.name or platform.system() == SystemPlatform.Linux.name:
```

Reply:
I have also made the necessary changes for this as well. Thanks for pointing out.